test/test_common/environment.h

```diff
@@ -21,6 +22,17 @@ class TestEnvironment {
   static void initializeOptions(int argc, char** argv);
 
   /**
+   * @return bool if testing only with IP type addresses only.
+   */
+  static bool isTestIpVersionOnly(const Network::Address::IpVersion& type);
```

Comment:
nit: this name is kind of strange. I would call it runTestForIpVersion() or something.